src/main/java/org/folio/circulation/resources/TitleLevelRequestNoticeSender.java

```diff
@@ -34,8 +34,14 @@ public Result<RequestAndRelatedRecords> sendNoticeOnRequestCreated(
     if (tlrSettings.isTitleLevelRequestsFeatureEnabled()
       && tlrSettings.getConfirmationPatronNoticeTemplateId() != null) {
 
-      PatronNoticeEvent requestCreatedEvent = createPatronNoticeEvent(request, eventType);
-      applyTlrConfirmationNotice(request.getTlrSettingsConfiguration(), requestCreatedEvent);
+      locationRepository.loadCampus(request.getItem().getLocation())
+        .thenCompose(r -> r.after(locationRepository::loadInstitution))
+        .thenApply(r -> r.map(request.getItem()::withLocation))
+        .thenApply(r -> r.map(request::withItem))
+        .thenAccept(r -> r.after(result -> {
```

Comment:
which type is `result`? Maybe we can give it a more descriptive name?

Reply:
fixed